.github/pull_request_template.md

```
@@ -0,0 +1,52 @@
+
+## Screenshots / Logs
+
+<!--
+Paste relevant terminal output, `kubectl describe` output,
+or screenshots that help reviewers verify the change.
+-->
+
+<details>
+<summary>Output</summary>
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)